custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -134,28 +134,28 @@ def _init_mlops(self):
         if self._params.get("model_id", None):
             self._mlops.set_model_id(self._params["model_id"])
 
+        self._configure_mlops()
+
         if self.supports_chat():
-            self._configure_mlops_for_chat()
-        else:
-            self._configure_mlops_for_non_chat()
+            self._prompt_column_name = self.get_prompt_column_name()
+            logger.debug("Prompt column name: %s", self._prompt_column_name)
 
         self._mlops.init()
 
-    def _configure_mlops_for_chat(self):
+    def _configure_mlops(self):
         # If monitor_settings were provided (e.g. for testing) use them, otherwise we will
         # use the API spooler as the default config.
         if self._params.get("monitor_settings"):
             self._mlops.set_channel_config(self._params["monitor_settings"])
         else:
+            for required_param in ["external_webserver_url", "api_token"]:
+                if required_param not in self._params:
+                    raise ValueError(f"MLOps API spooler requires '{required_param}' parameter")
```

Comment:
"API spooler" --> "MLOps monitoring".  Ideally, the spooler will be determined by the env vars.